mmdet3d/models/backbones/h3d_backbone.py

```diff
@@ -0,0 +1,79 @@
+import torch
+from mmcv.runner import load_checkpoint
+from torch import nn as nn
+from torch.nn import functional as F
+
+from mmdet.models import BACKBONES, build_backbone
+
+
+@BACKBONES.register_module()
+class H3DBackbone(nn.Module):
```

Comment:
We may change `H3DBackbone` to `MultiPointNet2SASSG` since it can be used for other general purposes.

Reply:
We can try to design `MultiBackbones`  (e.g. 4 different backbones) or `RepeatBackbones` (e.g. 4 identical backbones) to cover this case.